website/repos-info.json

```diff
@@ -0,0 +1 @@
ter display","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/1292"},{"comments":1,"number":826,"title":"docs: Instructions for compiling and using go binary for different platforms","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/826"},{"comments":3,"number":821,"title":"[Proposal]: Monitor Routes","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/821"},{"comments":2,"number":745,"title":"Record some videos to demonstrate some basic operations on dashboard","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/745"},{"comments":11,"number":733,"title":"Operation experience optimization","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/733"},{"comments":14,"number":648,"title":"feat: Should apisix dashboard support multi-tenancy feature?","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/648"},{"comments":10,"number":578,"title":"Can I set up stream_proxy through the new version of apisix-dashboard? No interface seems to be found","htmlUrl":"https://github.com/apache/apisix-dashboard/issues/578"}]},"apache/apisix-ingress-controller":{"info":{"description":"ingress controller for K8s","Star":437,"Watch":41,"Fork":157},"issues":[{"comments":0,"number":880,"title":"e2e: custom registry","htmlUrl":"https://github.com/apache/apisix-ingress-controller/issues/880"},{"comments":1,"number":782,"title":"request help:  APISIX Ingress Annotation  Document","htmlUrl":"https://github.com/apache/apisix-ingress-controller/issues/782"},{"comments":21,"number":73,"title":"kubectl plugin support","htmlUrl":"https://github.com/apache/apisix-ingress-controller/issues/73"}]},"apache/apisix-helm-chart":{"info":{"description":"Apache APISIX Helm Chart","Star":90,"Watch":29,"Fork":67},"issues":[]},"apache/apisix-docker":{"info":{"description":"the docker for Apache APISIX","Star":338,"Watch":35,"Fork":227},"issues":[{"comments":0,"number":200,"title":"feat request: build dashboard docker image on travis ci ARM machine","htmlUrl":"https://github.com/apache/apisix-docker/issues/200"}]},"apache/apisix-java-plugin-runner":{"info":{"description":"APISIX Plugin Runner in Java","Star":57,"Watch":25,"Fork":40},"issues":[{"comments":1,"number":92,"title":"request help: How to get all the information in the original request header.","htmlUrl":"https://github.com/apache/apisix-java-plugin-runner/issues/92"},{"comments":5,"number":75,"title":"feature: java plugin log path error","htmlUrl":"https://github.com/apache/apisix-java-plugin-runner/issues/75"},{"comments":3,"number":49,"title":"[discuss]: increase the timeout of socket communication between apisix and java plugin.","htmlUrl":"https://github.com/apache/apisix-java-plugin-runner/issues/49"}]},"apache/apisix-go-plugin-runner":{"info":{"description":"Go Plugin Runner for APISIX","Star":66,"Watch":27,"Fork":29},"issues":[]},"apache/apisix-python-plugin-runner":{"info":{"description":"Apache APISIX Python plugin runner","Star":15,"Watch":20,"Fork":13},"issues":[]}}
```

Comment:
And, maybe we can change each data value to 0 default.

Reply:
**Each** prop? I will rewrite this part manually.😅

Reply:
I mean to generate an empty data. Both info values and issue lists can be written as empty lists.